app/src/components/blocks/_lloydGeorge/lloydGeorgeDownloadComplete/LloydGeorgeDownloadComplete.tsx

```diff
@@ -1,32 +1,68 @@
 import React, { Dispatch, SetStateAction, useEffect } from 'react';
 import { Button, Card } from 'nhsuk-react-components';
-import { LG_RECORD_STAGE } from '../../../../types/blocks/lloydGeorgeStages';
-import { DOWNLOAD_STAGE } from '../../../../types/generic/downloadStage';
 import ReducedPatientInfo from '../../../generic/reducedPatientInfo/ReducedPatientInfo';
 import { focusLayoutDiv } from '../../../../helpers/utils/manageFocus';
 import useTitle from '../../../../helpers/hooks/useTitle';
+import DocumentsListView from '../../../generic/documentsListView/DocumentsListView';
+import { SearchResult } from '../../../../types/generic/searchResult';
+import { GenericDocument } from '../../../../types/generic/genericDocument';
+import { LG_RECORD_STAGE } from '../../../../types/blocks/lloydGeorgeStages';
+import { DOWNLOAD_STAGE } from '../../../../types/generic/downloadStage';
 
 export type Props = {
     setStage: Dispatch<SetStateAction<LG_RECORD_STAGE>>;
     setDownloadStage: Dispatch<SetStateAction<DOWNLOAD_STAGE>>;
     deleteAfterDownload: boolean;
+    numberOfFiles: number;
+    selectedDocuments?: Array<string>;
+    searchResults?: Array<SearchResult>;
 };
 
-function LloydGeorgeDownloadComplete({ setStage, setDownloadStage, deleteAfterDownload }: Props) {
+function LloydGeorgeDownloadComplete({
+    setStage,
+    setDownloadStage,
+    deleteAfterDownload,
+    numberOfFiles,
+    selectedDocuments,
+    searchResults,
+}: Props) {
     // temp solution to focus on layout div so that skip-link can be selected.
     // we should remove this when this component become a separate route.
     useEffect(() => {
         focusLayoutDiv();
     }, []);
 
+    const selectedFilesDownload = !!selectedDocuments?.length;
+    const pageHeader = 'Download complete';
+    useTitle({ pageTitle: pageHeader });
+
     const handleReturnButtonClick = () => {
         setStage(LG_RECORD_STAGE.RECORD);
         if (deleteAfterDownload) {
             setDownloadStage(DOWNLOAD_STAGE.REFRESH);
         }
     };
-    const pageHeader = 'Download complete';
-    useTitle({ pageTitle: pageHeader });
+
+    const documentsList = searchResults
+        ?.filter((document) => selectedDocuments?.includes(document.ID))
+        .map((document) => {
+            return {
+                ref: document.ID,
+                id: document.ID,
+                fileName: document.fileName,
+            };
+        }) as GenericDocument[];
+
+    const getCardHeader = () => {
+        if (selectedFilesDownload) {
+            return 'You have downloaded files from the record of:';
+        } else if (selectedDocuments) {
+            return 'You have downloaded the record of:';
```

Comment:
Can you wrap this in <p> tag so that you can provide a unique data-id for the messages?